bin/reth/tests/commands/bitfinity_reset_evm_state_it.rs

```diff
@@ -113,6 +113,8 @@ async fn bitfinity_test_reset_should_extract_all_accounts_data() {
         import_data.provider_factory.clone(),
         executor.clone(),
         parallel_requests,
+        750_000,
```

Comment:
in the cli default value we use 500000. Is this 750_000 meant to be correct?

Reply:
yes, it doesn't matter for tests